portfolio/src/main/webapp/script.js

```diff
@@ -12,39 +12,70 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/** @param slideIndex slide currently being displayed */
 var slideIndex = 1;
+
+/**
+ * Supported class states for dots and slides.
+ * @enum {string}
+ */
+const states = {
+  SHOW_DOT: ' active',
```

Comment:
is there a reason a whitespace is needed before the word "active"?

Reply:
The whitespace is necessary as I am attempting to add an attribute to the `"dot"` class turning it into `"dot active"`.

Reply:
That makes sense. What I would do is either add a comment to describe this as it is not obvious from first looking at it. Or you could even make it an enum called ADD_ATTRIBUTE or create a const string called ADD_ATTRIBUTE and then SHOW_DOT: ADD_ATTRIBUTE + 'active'

Reply:
Done  #8